gui/slick/js/core.js

```diff
@@ -2510,12 +2510,15 @@ var SICKRAGE = {
             $('body').on('click', '.epManualSnatch', function(event){
                 event.preventDefault();
                 var link = this;
-                $.getJSON(this.href, 
+                $(link).children('img').attr('src', srRoot + '/images/loading16.gif');
+                $.getJSON(this.href,
```

Comment:
@OmgImAlexis I was tempted to use a more complete $.ajax(). Because then I could also use timeout. But from the tests I was able to do, this seemed sufficient.